stone/api.py

```diff
@@ -96,7 +96,7 @@ class ApiNamespace(object):
     """
 
     def __init__(self, name):
-        # type: (str) -> None
+        # type: (typing.Text) -> None
```

Comment:
Just thinking out loud here, but are there other places we use `# type: (str)` where we should be using `# type: (typing.Text)`?